NEWS

```diff
@@ -126,6 +129,9 @@ Changed Functionality
 Removed Functionality
 ---------------------
 
+- The "segment_profiling" functionality and ``load_sample`` event have been removed
```

Comment:
This doesn't need to be duplicated. Adding it to the Breaking Changes section is enough.